drivers/espi/Kconfig.it8xxx2

```diff
@@ -125,4 +125,9 @@ config ESPI_IT8XXX2_PNPCFG_DEVICE_KBC_MOUSE
 	  With this option enabled, EC will send IRQ12 signal to host when the
 	  KBC mouse output buffer is full.
 
+config ESPI_SET_VW_VALID_WITH_LEVEL
```

Comment:
Is this Kconfig needed?  Should the valid bit always be set for supported virtual wires?

I see at least one other [driver](https://github.com/zephyrproject-rtos/zephyr/blob/392a753ac20eb46ba782b5484b2028438f9d9e5e/drivers/espi/espi_npcx.c#L1249) unconditionally sets the valid bit for all virtual wires at init time.

Reply:
Yes, sir. 
Indeed, it should be set for all the virtual wires, I'll removed the Kconfig option.